### PR TITLE
Simple code refactor: functions logic, repetitive messages, throws catching. 

### PR DESCRIPTION
**Pull request by Jhooomn**
updated: it was used sonar-lint plugin for this pull request; mainly purpose is to reduce complexity or redundant piece of code inside classes... for example like:

- common string messages
- didn't use assertThat functions and create same logic
- catch throwable instead of exceptions

### CODE REVIEW

**Comment by violetagg** [comment]
@Jhooomn Please create PR for every type of issue i.e. one handling extraction of the strings, the other exception vs throwable etc.

**Comment by Jhooomn** [comment]
@violetagg  I'll keep on mind for next PR's. Could you merge this one?

File: reactor-netty-core/src/jarFileTest/java/reactor/netty/JarFileShadingTest.java [hunk 1]
```diff
@@ -39,20 +40,22 @@
  */
 class JarFileShadingTest extends AbstractJarFileTest {
 
+	public static final String REACTOR_STRING = "reactor";
```
Comment (violetagg) [inline reactor-netty-core/src/jarFileTest/java/reactor/netty/JarFileShadingTest.java:43]:
```suggestion
	static final String REACTOR_STRING = "reactor";
```

File: reactor-netty-core/src/jarFileTest/java/reactor/netty/JarFileShadingTest.java [hunk 2]
```diff
@@ -64,6 +67,7 @@ void testManifestContent() throws IOException {
 		ZipFile jar = new ZipFile(jarFilePath.toString());
 		ZipEntry manifest = jar
 				.stream()
+				.filter(ze -> Objects.nonNull(ze) && Objects.nonNull(ze.getName()))
```
Comment (violetagg) [inline reactor-netty-core/src/jarFileTest/java/reactor/netty/JarFileShadingTest.java:70]: I'm not sure that this check is necessary. Wdyt?

File: reactor-netty-core/src/main/java/reactor/netty/FutureMono.java [hunk 3]
```diff
@@ -99,7 +99,7 @@ public void subscribe(CoreSubscriber<? super Void> s) {
 			try {
 				f = deferredFuture.get();
 			}
-			catch (Throwable t) {
+			catch (Exception t) {
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/FutureMono.java:102]: catching `Throwable` is deliberate

File: reactor-netty-core/src/main/java/reactor/netty/channel/AbortedException.java [hunk 4]
```diff
@@ -26,6 +26,7 @@
  */
 public class AbortedException extends RuntimeException {
 	static final String CONNECTION_CLOSED_BEFORE_SEND = "Connection has been closed BEFORE send operation";
+	public static final String CONNECTION_RESET_BY_PEER_MESSAGE = "Connection reset by peer";
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/AbortedException.java:29]:
```suggestion
	static final String CONNECTION_RESET_BY_PEER_MESSAGE = "Connection reset by peer";
```

File: reactor-netty-core/src/main/java/reactor/netty/channel/ChannelOperationsHandler.java [hunk 5]
```diff
@@ -76,7 +76,7 @@ final public void channelInactive(ChannelHandlerContext ctx) {
 				listener.onStateChange(connection, ConnectionObserver.State.DISCONNECTING);
 			}
 		}
-		catch (Throwable err) {
+		catch (Exception err) {
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/ChannelOperationsHandler.java:79]: catching `Throwable` is deliberate

File: reactor-netty-core/src/main/java/reactor/netty/channel/ChannelOperationsHandler.java [hunk 6]
```diff
@@ -108,7 +108,7 @@ final public void channelRead(ChannelHandlerContext ctx, Object msg) {
 				ReferenceCountUtil.release(msg);
 			}
 		}
-		catch (Throwable err) {
+		catch (Exception err) {
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/ChannelOperationsHandler.java:111]: catching `Throwable` is deliberate

File: reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java [hunk 7]
```diff
@@ -268,12 +271,12 @@ final void drainReceiver() {
 				try {
 					if (logLeakDetection.isDebugEnabled()) {
 						if (v instanceof ByteBuf) {
-							((ByteBuf) v).touch(format(channel, "Receiver " + a.getClass().getName() +
-									" will handle the message from this point"));
+							((ByteBuf) v).touch(format(channel, RECEIVER_MESSAGE + a.getClass().getName() +
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java:274]: In production where DEBUG is not enabled, these strings will not be created (guarded by `logLeakDetection.isDebugEnabled()`). If we convert these strings to a constants, then these constants will be created always regardless whether it is needed or not. Wdyt?

File: reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java [hunk 8]
```diff
@@ -461,7 +464,6 @@ final void terminateReceiver(@Nullable Queue<?> q, CoreSubscriber<?> a) {
 		Throwable ex = inboundError;
 		receiver = null;
 		if (ex != null) {
-			//parent.listener.onReceiveError(channel, ex);
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java:464]: Please return this commented line

File: reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java [hunk 9]
```diff
@@ -282,7 +285,7 @@ else if (v instanceof ByteBufHolder) {
 					try {
 						ReferenceCountUtil.release(v);
 					}
-					catch (Throwable t) {
+					catch (Exception t) {
```
Comment (violetagg) [inline reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java:288]: catching `Throwable` is deliberate

**Review by violetagg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**